script/playercomponent.js

```diff
@@ -228,37 +250,41 @@ define(
       function attemptCdnFailover (errorProperties, bufferingTimeoutError) {
         var shouldFailover = MediaResilience.shouldFailover(mediaMetaData.urls.length, getDuration(), getCurrentTime(), getLiveSupport(device), windowType, transferFormat);
 
+        var failover = function (time) {
+          cdnFailover(time, thenPause, errorProperties, bufferingTimeoutError);
+        };
+        var errorCallback = function () {
+          bubbleFatalError(errorProperties, bufferingTimeoutError);
+        };
+
         if (shouldFailover) {
           var thenPause = playbackStrategy.isPaused();
           tearDownMediaElement();
 
           var failoverTime = getCurrentTime();
-          if (transferFormat === TransferFormats.HLS && LiveSupportUtils.needToGetManifest(windowType, getLiveSupport(device))) {
-            manifestReloadFailover(failoverTime, thenPause, errorProperties, bufferingTimeoutError);
-          } else {
-            cdnFailover(failoverTime, thenPause, errorProperties, bufferingTimeoutError);
-          }
+          reloadManifest(failoverTime, failover, errorCallback);
         } else {
-          bubbleFatalError(errorProperties, bufferingTimeoutError);
+          errorCallback();
         }
       }
 
-      function manifestReloadFailover (failoverTime, thenPause, errorProperties, bufferingTimeoutError) {
-        ManifestLoader.load(
-          bigscreenPlayerData.media.urls,
-          bigscreenPlayerData.serverDate,
-          {
-            onSuccess: function (manifestData) {
-              var windowOffset = manifestData.time.windowStartTime - getWindowStartTime();
-              bigscreenPlayerData.time = manifestData.time;
-              failoverTime -= windowOffset / 1000;
-              cdnFailover(failoverTime, thenPause, errorProperties, bufferingTimeoutError);
-            },
-            onError: function () {
-              bubbleFatalError(errorProperties, bufferingTimeoutError);
+      function reloadManifest (time, successCallback, errorCallback) {
+        if (transferFormat === TransferFormats.HLS && LiveSupportUtils.needToGetManifest(windowType, getLiveSupport(device))) {
+          ManifestLoader.load(
+            bigscreenPlayerData.media.urls,
+            bigscreenPlayerData.serverDate,
+            {
+              onSuccess: function (manifestData) {
+                var windowOffset = manifestData.time.windowStartTime - getWindowStartTime();
+                bigscreenPlayerData.time = manifestData.time;
```

Comment:
An existing bug, but we really need to propagate the updated time data to the playback strategy. I think @jlks and @matt-stephenson are looking into this so maybe we can live with it for this PR if it doesn't break fake seeking?